include/ck/tensor_operation/gpu/device/device_batched_gemm_softmax_gemm_permute.hpp

```diff
@@ -87,8 +87,8 @@ template <index_t NumDimG,
           MaskingSpecialization MaskingSpec>
 struct DeviceBatchedMultiheadAttentionForward : public BaseOperator
 {
-    static constexpr index_t NumAcc0Bias = Acc0BiasDataType::Size();
-    static constexpr index_t NumAcc1Bias = Acc1BiasDataType::Size();
+    static constexpr index_t NumAcc0Bias = 1;
+    static constexpr index_t NumAcc1Bias = 0;
 
```

Comment:
Should the two lines be removed since they are not used ?

Reply:
done